packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -224,6 +242,14 @@ class ToOOXMLVisitor {
                             this.globalOOXML += ooxml;
                             this.tags = [];
                         } else {
+                            if (this.getClass(subNode) === TRANSFORMED_NODES.link) {
+                                const relationshipTag = `<Relationship Id="rId${
+                                    this.relationships.length + 3
```

Comment:
Why is `3` used here? You used `2` above. It is better to hardcode that offset and then reuse it everywhere.

Reply:
@algomaster99 
At one moment, the length is zero. After that the relationship tag is append. This makes the length increase by 1. Hence, the difference.

Reply:
So it would be better to use `RELATIONSHIP_OFFSET +1`.